composer.json

```
@@ -44,7 +44,7 @@
   "scripts": {
     "test": "phpunit",
     "test-coverage": "vendor/bin/phpunit --coverage-html ./coverage",
-    "cs-fixer": "vendor/bin/php-cs-fixer fix"
+    "lint": "php-cs-fixer fix --diff --dry-run"
```

Let's run `php-cs-fixer` from vendor folder to avoid any version conflicts
```suggestion
    "lint": "vendor/bin/php-cs-fixer fix --diff --dry-run"
```

Also can we leave `cs-fixer` command as it is and add `lint` as a new command.

---

Thanks for the feedback, updated.